pkg/usersettings/types.go

```diff
@@ -0,0 +1,20 @@
+package usersettings
+
+type ResourceData struct {
+	Username string
+	UID      string
+	// The resource suffix uses the User uid like "-12341234-1234-1234-1234-123412341234" or "" for kube:admin
+	ResourceSuffix string
```

Comment:
Since this is not an suffix in case of the Role and RoleBinding I would name it `ResourceIdentifier`. Maybe?

Reply:
Renamed.

Alternative, I could create one type like this:

```go
type resourceIdentifier struct {
	userName string
	configMapName string
	roleName string
	roleBindingName string
}
```

Wdyt? I like the idea.. :thinking: